### ISSUE TITLE
Add filetype to import text as protoflux string input

### ISSUE THREAD

**Issue by Lexevolution** [Yellow-Dog-Man/Resonite-Issues#1754]
### Is your feature request related to a problem? Please describe.

Copying large amounts and multiple files worth of text in to Resonite can be tedious work, and I'd like a solution that would also make it easy to connect this text to protoflux.

### Describe the solution you'd like

A file extension that would allow any text file to import as a protoflux string input, which would let me also batch import multiple text files in to multiple string input nodes.

### Describe alternatives you've considered

- Importing the text files directly, then referencing the text uix field in the generated uix as a source (this is slower to set up)
- Copying and pasting each text file in to its own string input node (This is also slower to set up)

### Requesters

Lexevo

**Comment by epicEaston197** [comment]
In my opinion i think this will be suitable for the text file to be imported as a data model store because when you are importing things like G Code which are incredibly big text files to render and would crash the game if they do so

**Comment by Lexevolution** [comment]
@shiftyscales I did know about the batch importing, however, I did not know you could use the text canvas as a reference to the text itself! This would work fine for me, but on the other hand, it's always nice to save a few minutes for the ~60 text files I wanted to import.

**Comment by lxw404** [comment]
What seems to be asked for here is an analog to what exists another platform; there was a specific format for importing strings called `.logixstring`, which when a file of this extension was dropped it created a data model store node with the actual text in it.  Something similar like `.protofluxstring` would be the equivalent here, so that **very large** text files (such as entire novels, or gcode as mentioned above) could be dropped and stored directly in a protoflux node.  I do agree that the generic modular importer/exporter would be more useful though.

**Comment by Lexevolution** [comment]
Yes, @lxw404 is correct. I refrained from saying this, and how it worked, due to the current issue guidelines. In previous versions of FrooxEngine, there was a filetype that put text directly in to a string variable node, and also completely skipped the import dialogue. I was confused as to why you kept mentioning the import dialogue, since batch importing files would only show it once anyways.
